shared/cluster.go

```diff
@@ -47,21 +43,32 @@ func ManageWorkload(action string, workloads ...string) error {
 		return ReturnLogError("invalid action: %s. Must be 'apply' or 'delete'", action)
 	}
 
-	resourceDir := BasePath() + "/workloads/" + Arch
-
-	files, err := os.ReadDir(resourceDir)
+	prod, _, err := Product()
 	if err != nil {
-		return ReturnLogError("Unable to read resource manifest file for: %s\n", resourceDir)
+		return err
+	}
+
+	if envErr := config.SetEnv(BasePath() + fmt.Sprintf("/config/%s.tfvars",
+		prod)); envErr != nil {
+		return ReturnLogError("error setting env: %w\n", envErr)
+	}
+	arch := os.Getenv("arch")
+
+	resourceDir := BasePath() + "/workloads/" + arch
```

Comment:
oh please no ! lol 

i added like this to prevent adding a "random" param on the signature, keeping this lighter , since this function its called a lot of times everywhere 
ans this is not a optional var today, so we are always going to have this env arch